resources/dashboards/Ports-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "title": "Ports Dashboard",
+   "hits": 0,
+   "description": "",
```

Comment:
If it is not a big deal to add a description, I think it would be helpful to put something here about non-ephemeral ports.